app/components/version-list/row.hbs

```diff
@@ -20,7 +20,9 @@
         {{@version.releaseTrack}}
       {{/if}}
 
-      <EmberTooltip @text={{this.releaseTrackTitle}} @side="right" data-test-release-track-title/>
+      <EmberTooltip @side="right" data-test-release-track-title>
+        {{this.releaseTrackTitle}}
+      </EmberTooltip>
```

Comment:
I assume this is faster because `this.releaseTrackTitle` is now only evaluated lazily when the tooltip is rendered, instead of being evaluated eagerly because it was passed as an argument to the component?

Reply:
You're absolutely right, that's exactly what this code snippet does. No worries, I usually try to follow that approach as well. In this case, I could split it into two commits for better clarity.